layouts/html.hbs

```diff
@@ -6,6 +6,14 @@
       if (window.name == 'overlay') {
         window.isOverlay = true;
       }
+      function onDocumentReady(cb) {
+        if (document.readyState === "complete"
```

Comment:
Is it possible we will ever invoke `onDocumentReady` when the document is already loaded? I'm wondering if we need this condition or if the `addEventListener` will suffice.

Reply:
I'm using it inside the iframeResizer onMessage, so we do need to check the ready state. Also I think it makes it a little more robust overall